trade/trader_stakecube.js

```diff
@@ -300,7 +303,10 @@ module.exports = (
      * @param {Number} coin1Amount Base coin amount. Provide either coin1Amount or coin2Amount.
      * @param {Number} limit StakeCube supports only limit orders
      * @param {Number} coin2Amount Quote coin amount. Provide either coin1Amount or coin2Amount.
-     * @returns {Promise<unknown>|undefined}
+     * @returns {Promise<{orderId: string|boolean, message: string}>|message: string}
```

Comment:
Why?
Sample returns:
- string: `Some error has occurred`
- object: `{orderId: '12345', message: 'Order placed'}`
- object: `{orderId: false, message: 'Some error has occurred'}`

Reply:
Sorry, I miss understood the typing because of GitHub's highlighting.

In this case, it should be `| string`, not `| message: string`

```ts
Promise<...> | string
```

Reply:
It might be a good idea to also add a description that the function returns a string in case of an error.

Reply:
> It might be a good idea to also add a description that the function returns a string in case of an error.

Already added, see below:

     *   In case of pre-request error, returns error message string.
     *   After request, returns an object with orderId and message. Cast orderId to string.
     *   In case if order was not placed, orderId is false, and message contains error info.